packages/extension-polkagate/src/util/workers/sharedWorker.js

```
@@ -9,49 +11,56 @@ import { getAssetOnRelayChain } from './shared-helpers/getAssetOnRelayChain.js';
 
 const assetsChains = createAssets();
 
-onmessage = (e) => {
-  const { functionName, parameters } = e.data;
+// Handle connections to the shared worker
+onconnect = (event) => {
```

Biome (1.9.4)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)